src/main/java/com/github/dockerjava/core/util/CertificateUtils.java

```diff
@@ -41,115 +49,157 @@ public static boolean verifyCertificatesExist(String dockerCertPath) {
         return result;
     }
 
+    /**
+     * @param dockerCertPath with standard named files.
+     */
     public static KeyStore createKeyStore(final String dockerCertPath) throws NoSuchAlgorithmException,
             InvalidKeySpecException, IOException, CertificateException, KeyStoreException {
-        KeyPair keyPair = loadPrivateKey(dockerCertPath);
-        List<Certificate> privateCertificates = loadCertificates(dockerCertPath);
+        return createKeyStore("key.pem", "cert.pem");
```

Comment:
Smells like i missed this wrapper when refactored methods to be reusable for non-file based usage.
Could you fix or should i?